web/src/components/pages/contribution/speak/speak.tsx

```diff
@@ -62,20 +59,41 @@ enum RecordingError {
 const UnsupportedInfo = () => (
   <div className="empty-container">
     <div className="error-card card-dimensions unsupported">
-      <Localized id="record-platform-not-supported" />
-      <p className="desktop">
-        <Localized id="record-platform-not-supported-desktop">
-          <span />
-        </Localized>
-      </p>
-      <div>
-        <a rel="noopener noreferrer" target="_blank" href="https://www.firefox.com/" title="Firefox">
-          <FirefoxColor />
-        </a>{' '}
-        <a rel="noopener noreferrer" target="_blank" href="https://www.google.com/chrome" title="Chrome">
-          <ChromeColor />
-        </a>
-      </div>
+      {isIOS() && !isMobileSafari() ? (
+        <>
+          <BalanceText>
```

Comment:
This is cool, do you know if this supports RTL languages? If yes, we should use this for the overall sentence display cards too.

Reply:
I wrote a quick custom test with `direction: rtl` on [the library's storybook](https://khan.github.io/react-balance-text/?selectedKind=BalanceText&selectedStory=Basic%20Text%20Balancing&full=0&down=1&left=1&panelRight=0&downPanel=kadirahq%2Fstorybook-addon-actions%2Factions-panel) and it seemed to work 👍